src/joinboost/executor.py

```diff
@@ -335,6 +329,7 @@ def _execute_query(self, q):
             print(e)
         return result
 
+
 class PandasExecutor(DuckdbExecutor):
     # Because Pandas is not a database, we use a dictionary to store table_name -> dataframe
     table_registry = {}
```

Comment:
For the init, maybe we don't need table_counter, and "prefix = 'joinboost_'" is in the super init